CMakeLists.txt

```diff
@@ -46,3 +46,11 @@ include(CTest)
 
 add_subdirectory(k4ActsTracking)
 add_subdirectory(test)
+
+install(EXPORT ${PROJECT_NAME}Targets
+  NAMESPACE ${PROJECT_NAME}::
+  FILE "${PROJECT_NAME}Targets.cmake"
+  DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}/"
+  )
+
+gaudi_install(CMAKE cmake/${PROJECT_NAME}Config.cmake)
```

Comment:
The `cmake/k4ActsTrackingConfig.cmake` that gaudi is trying to install here is missing at the moment. I am not sure how `k4ActsTracking` would be consumed downstream without it.

In k4FWCore, we create it via `configure_package_config_file`, [like so](https://github.com/key4hep/k4FWCore/blob/main/cmake/k4FWCoreCreateConfig.cmake). I suppose something similar would need to happen here as well, or at least as a minimal version a usable `Config.cmake` file would need to be placed into the `cmake` folder.

Reply:
Ah, forgot to add the `cmake/k4ActsTrackingConfig.cmake` file. I don't think we need the entire infrastructure of https://github.com/key4hep/k4FWCore/pull/230 in this PR (but I can add it in a followup), so I'm goingt o add the local version that is more similar to https://github.com/key4hep/k4FWCore/blob/v01-01/cmake/k4FWCoreConfig.cmake

Reply:
Should be ready now. Fesolving.